demo/demo.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#Test oc tool
+#Assumes User is logged in to cluster
+
+path=$1
+url=$2
+
+# Will create spring rest deployment
+cd $path/basic-nginx
+ansible-galaxy install -r requirements.yml --roles-path=galaxy
+ansible-playbook -i ./.applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e skip_manual_promotion=true -e source_code_url=$url
+
+
+# Wait a bit (5 min) for the app to be finished deployed
+sleep 300
```

Comment:
Instead of silently waiting for 5 minutes, let's have it wait for the user to "continue" by pressing Enter.